R/calculate_capCosts.R

```diff
@@ -15,50 +14,40 @@ calculate_capCosts <-function(base_price, Fdemand_ES,
                               EDGE2CESmap,
                               EDGE2teESmap,
                               REMINDyears,
-                              scenario,
-                              REMIND2ISO_MAPPING=NULL){
+                              scenario){
 
-  teEs <- iso <- variable <- value <- demand_F <- `.` <- subsector_L3 <- vehicle_type <- NULL
-  vehicles_number <- annual_mileage <- load_factor <- demand <- NULL
+  teEs <- region <- variable <- value <- demand_F <- `.` <- subsector_L3 <- vehicle_type <- NULL
+  vehicles_number <- annual_mileage <- load_factor <- demand <- technology <- NULL
 
   ## the non fuel price is to be calculated only for motorized entries
   Fdemand_ES=Fdemand_ES[!subsector_L3 %in% c("Walk","Cycle"),]
   base_price=base_price[!subsector_L3 %in% c("Walk","Cycle"),]
   ## merge prices and demand
-  data=merge(base_price,Fdemand_ES,all.y=TRUE,by=intersect(names(base_price),names(Fdemand_ES)))
+  data=merge(base_price,Fdemand_ES[technology != "Hybrid Electric"],all.y=TRUE,by=intersect(names(base_price),names(Fdemand_ES)))
```

Comment:
maybe comment on exclusion of hybrid electric

Reply:
Included comment, there is a bug is on my todo list behind the exclusion of the Hybrid Electric. I will look into that.